modules/prov/src/test/resources/data/nexus/prov/activity/v0.1.0/ordering-activity1-PASS-c30-ignored.json

```diff
@@ -0,0 +1 @@
+[{"@id":"http://example.com/Graduation"},{"@id":"http://example.com/doctor_smith","@type":["http://www.w3.org/ns/prov#Entity"]},{"@id":"http://example.com/graduation","@type":["http://www.w3.org/ns/prov#Activity","http://example.com/Graduation"],"http://www.w3.org/ns/prov#startedAtTime":[{"@value":"2012-04-15T13:00:00-04:00","@type":"http://www.w3.org/2001/XMLSchema#dateTime"}],"http://www.w3.org/ns/prov#used":[{"@id":"http://example.com/ms_smith"}],"http://www.w3.org/ns/prov#generated":[{"@id":"http://example.com/doctor_smith"}],"http://www.w3.org/ns/prov#endedAtTime":[{"@value":"2012-04-15T14:30:00-04:00","@type":"http://www.w3.org/2001/XMLSchema#dateTime"}]},{"@id":"http://example.com/ms_smith","@type":["http://www.w3.org/ns/prov#Entity"]},{"@id":"http://www.w3.org/ns/prov#Activity"},{"@id":"http://www.w3.org/ns/prov#Entity"}]
```

Comment:
Did you want to include this file in the PR?

Reply:
No. Removed